e2e/tests/test_e2e.py

```diff
@@ -395,12 +395,20 @@ def test_config_update(self):
             "spec": {
                 "postgresql": {
                     "parameters": {
-                        "max_connections": new_max_connections_value
+                        "max_connections": new_max_connections_value,
+                        "wal_level": "logical"
                      }
                  },
-                 "patroni": {
+                "users": {
+                    "test_user": []
+                },
+                "databases": {
+                    "foo": "test_user",
+                    "bar": "test_user"
+                },
+                "patroni": {
```

Comment:
```suggestion
                "patroni": {
```

Do not create users and databases here but use existing ones. e2e test uses the minimal manifest so we already have foo (owner: zalando) and bar (owner: bar_owner). With test_user we get an extra secret which leads to the next test failing.